src/panels/config/entities/ha-config-entities.ts

```diff
@@ -432,6 +419,9 @@ export class HaConfigEntities extends SubscribeMixin(LitElement) {
           ),
           unavailable,
           restored,
+          localized_platform:
+            this.hass.localize(`component.${entry.platform}.title`) ||
```

Comment:
Localize should also be passed to the memoized function here, instead of using `this.hass.localize`

Reply:
done